icons/origami.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "../icon.schema.json",
+    "contributors": [
+      "gurtt"
+    ],
+    "tags": [
+        "paper",
+        "bird"
```

Comment:
There are several linting issues with this file, can you make sure to format it in accordance with our prettier config?